src/Congo.WebApi/Data/CongoContext.cs

```diff
@@ -1,9 +1,11 @@
-﻿using Microsoft.EntityFrameworkCore;
+﻿using Congo.WebApi.Data.Models;
+using Microsoft.EntityFrameworkCore;
 
 namespace Congo.WebApi.Data
 {
     public class CongoContext : DbContext
     {
+        public DbSet<Product> Products { get; set; }
```

Comment:
I think we should follow standard C# structure here, which would be to put properties below the constructor.